sdk/eventhub/Azure.Messaging.EventHubs/tests/Primitives/PartitionReceiverTests.cs

```diff
@@ -0,0 +1,572 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Messaging.EventHubs.Consumer;
+using Azure.Messaging.EventHubs.Core;
+using Azure.Messaging.EventHubs.Diagnostics;
+using Azure.Messaging.EventHubs.Primitives;
+using Moq;
+using NUnit.Framework;
+
+namespace Azure.Messaging.EventHubs.Tests
+{
+    /// <summary>
+    ///   The suite of tests for the <see cref="PartitionReceiver" />
+    ///   class.
+    /// </summary>
+    ///
+    [TestFixture]
+    public class PartitionReceiverTests
+    {
+        /// <summary>
+        ///   Verifies functionality of the constructor.
+        /// </summary>
+        ///
+        [Test]
+        [TestCase(null)]
+        [TestCase("")]
+        public void ConstructorValidatesTheConsumerGroup(string consumerGroup)
+        {
+            var expectedType = consumerGroup is null ? typeof(ArgumentNullException) : typeof(ArgumentException);
```

Comment:
I would think it's enough to confirm that we're getting back an argument exception instance;  the distinction for `null` is the responsibility of the `Argument` class and doesn't really alter the experience for callers here.    That said, if you feel like keeping it, I'm not opposed.

Reply:
I have removed the `expectedType` thing from every test (not only the `PartitionReceiver` ones).